app/controllers/tracks_controller.rb

```diff
@@ -140,7 +140,7 @@ def transformed_attributes
 
     if attributes[:track_artists].present?
       attributes[:track_artists] = attributes[:track_artists].map do |ta|
-        TrackArtist.new(artist_id: ta[:artist_id], name: ta[:name], role: ta[:role], order: ta[:order] || 0)
+        TrackArtist.new(artist_id: ta[:artist_id], name: ta[:name], role: ta[:role], order: ta[:order] || 0, hidden: ta[:hidden])
```

Comment:
Does this require clients to pass the `hidden` parameter? What happens if they don't? (Maybe add some tests to confirm the expected behavior?)

Reply:
It would have been `false`, since that was set as default. Since I also added `null: false` to the column, I had to add a fallback value